tests/scripts/task_lint.sh

```diff
@@ -25,48 +25,69 @@ cleanup()
 trap cleanup 0
 
 
-echo "Convert scripts to Python..."
-tests/scripts/task_convert_scripts_to_python.sh
+# These shards are solely for CI to enable the lint job to have some parallelism.
 
-# TODO: Remove this ad-hoc pip install once https://github.com/apache/tvm/pull/10741
-# is added to the ci_lint Docker image
-python3 -m pip install --user -r jenkins/requirements.txt
-echo "Check Jenkinsfile generation"
-python3 jenkins/generate.py --check
+function shard1 {
+  echo "Convert scripts to Python..."
+  tests/scripts/task_convert_scripts_to_python.sh
 
-echo "Checking file types..."
-python3 tests/lint/check_file_type.py
+  # TODO: Remove this ad-hoc pip install once https://github.com/apache/tvm/pull/10741
```

Comment:
I think this has now landed?

Reply:
Yup you're right, we don't need this anymore